python/pip_install/parse_requirements_to_bzl/__init__.py

```diff
@@ -119,6 +119,14 @@ def main() -> None:
         required=True,
         help="Path to fully resolved requirements.txt to use as the source of repos.",
     )
+    parser.add_argument(
+        "--python_interpreter",
+        help="The python interpreter that will be used to download and unpack the wheels.",
+    )
+    parser.add_argument(
+        "--python_interpreter_target",
+        help="Bazel target of the python interpreter. It will take precedence over python_interpreter.",
```

Comment:
Some documentation explaining that this has to be a reference to another external repository would be helpful. To me the name implies that this could be a reference to a target in the current workspace, which wouldn't work.

Reply:
That's right thanks, outside of the `.bzl` this becomes much less obvious, I've updated the documentation. I've not changed the name of the argument to keep it consistent with `whl_library`/`pip_parse` though. (sorry for the slow reaction, I was away from computers!).